package.json

```diff
@@ -21,7 +21,8 @@
     "setup": "npm install",
     "svgo": "bash ./tasks/optimize-svg.sh",
     "lint:js": "eslint ./*.js app config lib server tests",
-    "test": "ember test"
+    "test": "ember test",
+    "nuke": "npm cache clean --force && rm -rf node_modules/ && rm package-lock.json"
```

Comment:
is it still needed?

Reply:
Yeah, it's a script that we'll be using in the future

Reply:
then why don't you call `npm run nuke` in dockerfile?

Reply:
then why don't you call `npm run nuke` in dockerfile?

Reply:
It was causing an error when deploying to sandbox.
Considering that we'll be removing this code next week anyway I don't really think that this matters in the long run.